keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverService.kt

```diff
@@ -50,11 +54,14 @@ interface CloudDriverService {
   @GET("/credentials")
   fun listCredentials(): Set<Credential>
 
+  @GET("/credentials/{account}")
+  fun getCredential(@Path("account") account: String) : Credential
+
   @GET("/{provider}/loadBalancers/{account}/{region}/{name}")
-  fun getElasticLoadBalancer(
+  fun getLoadBalancer(
     @Path("provider") provider: String,
     @Path("account") account: String,
     @Path("region") region: String,
     @Path("name") name: String
-  ): Set<ElasticLoadBalancer>
```

Comment:
`ElasticLoadBalancer` is AWS terminology, we should use cloudprovider-agnostic terms when fetching direct from clouddriver.